waspc/data/Generator/templates/server/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "server",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "nodemon ./src/server.js",
+    "debug": "DEBUG=server:* npm start",
+    "standard": "standard"
+  },
+  "engines": {
+    "node": "^14.0.0",
+    "npm": "^6.0.0"
+  },
+  "dependencies": {
```

Comment:
I didn't know about it, I looked it up (you lazy boy) and it says: `Helmet helps you secure your Express apps by setting various HTTP headers. It's not a silver bullet, but it can help!`. Cool!
My idea was to just go with what we have now, and then we can improve upon it -> there are also many other things that could be improved.
I made an issue for this here: https://github.com/wasp-lang/wasp/issues/22 .